ui/src/app/shared/components/badge-panel/badge-panel.tsx

```diff
@@ -20,6 +20,9 @@ export const BadgePanel = ({app, project}: {app?: string; project?: string}) =>
         let alt = '';
         if (app) {
             badgeURL = `${root}api/badge?name=${app}&revision=true`;
+            if (nsEnabled) {
+                badgeURL += `&namespace=${appNamespace}`
+            }
```

Comment:
I believe this check is not necessary; you can pass the namespace parameter regardless of whether apps-in-any-namespace is enabled. However, passing a namespace other than the installation namespace to an Argo CD that does not have apps-in-any-namespace enabled, will result in an error.